chart/templates/_helpers.yaml

```diff
@@ -995,3 +995,11 @@ capabilities:
             app: keda-operator
   {{- end }}
 {{- end }}
+
+{{- define "deployment_strategy" }}
+  {{- if and .strategy.type (eq .strategy.type "Recreate") }}
```

Comment:
If the user still want to use rolling update strategy and log.persistent = True, what u did here will prevent him to do so although it works.
Maybe u should check if the log is persistent u dont need to change the strategy.
Or maybe just change the default strategy.

Reply:
@romsharon98 Hey! How are you doing? Please, pay attention that `.strategy` exists and comes from default values. So, it just fixes the default value when it's Recreate type. Because the default values has an incompatible attribute (rollingUpdate) for.